packages/react/src/select-group-context/SelectGroupContext.ts

```diff
@@ -0,0 +1,6 @@
+import { createContext } from "@radix-ui/react-context";
+
+export const [SelectGroupContextProvider, useSelectGroupContext] =
+  createContext<{
+    id?: string;
+  }>("SelectGroup");
```

Comment:
we need to provide a default value here otherwise SelectLabel cannot be used outside of SelectGroup
```suggestion
  }>("SelectGroup", { id: undefined });
```

Reply:
handled in https://github.com/optimizely-axiom/optiaxiom/pull/540